features/account/managing_account/user_profile_validation.feature

```diff
@@ -0,0 +1,42 @@
+@managing_account
+Feature: User account profile validation
+    In order to avoid making mistakes when changing my personal information
+    As a logged user
+    I want to be prevented from changing fields to incorrect values
+
+    Background:
+        Given the store operates on a single channel in "France"
```

Comment:
USA would be more appropriate. Why does president of USA register in a store, which operates only in France?